### PR TITLE
Added geoWebDisplay command that uses jsroot to render the Geometry

### PR DESCRIPTION
**Pull request by bcouturi**
BEGINRELEASENOTES
- Added geoWebDisplay command that uses jsroot to render the Geometry, requires ROOT7 ROOTEve
ENDRELEASENOTES

I had to add ROOT::ROOTEve as a target to the plugin library. This should be done conditionally and I would be grateful for help on how to formulate that.

### CODE REVIEW

**Comment by andresailer** [comment]
Maybe you structure this similar to the GDMLPlugins?

https://github.com/AIDASoft/DD4hep/blob/5b5bdfdb64fe36b07b6b78a41c4db223e8f8d077/DDCore/CMakeLists.txt#L106-L117

**Comment by bcouturi** [comment]
Looking into the sources, would it be worth grouping this with DDEve ?
I notice it depends on ROOT::Eve, but not ROOT::ROOTEve

**Comment by andresailer** [comment]
ROOTEve is the ROOT7 library?
Fine for me to put this with DDEve

File: UtilityApps/CMakeLists.txt [hunk 1]
```diff
@@ -41,6 +41,12 @@ if(TARGET Geant4::Interface)
   LIST(APPEND OPTIONAL_EXECUTABLES dumpdetector)
 endif()
 
+# the web display tool depends on the ROOT7 version of DDEve
+if(TARGET ROOT::ROOTEve)
+  add_executable(geoWebDisplay src/webdisplay.cpp)
+  target_link_libraries(geoWebDisplay DD4hep::DDCore)
+endif()
```
Comment (andresailer) [inline UtilityApps/CMakeLists.txt:48]:
```suggestion
  LIST(APPEND OPTIONAL_EXECUTABLES geoWebDisplay)
endif()
```

File: UtilityApps/CMakeLists.txt [hunk 2]
```diff
@@ -78,6 +84,7 @@ add_executable(teveDisplay src/teve_display.cpp src/next_event_dummy.cpp G__eve.
 target_link_libraries(teveDisplay DD4hep::DDRec ROOT::Core ROOT::Eve ROOT::Gui ROOT::Graf3d ROOT::RGL )
 
 INSTALL(TARGETS geoDisplay
+  geoWebDisplay
```
Comment (andresailer) [inline UtilityApps/CMakeLists.txt:87]:
```suggestion
```

**Review by petricm — APPROVED** [review]

**Pull request merged**